pulsar-functions/instance/src/main/python/python_instance.py

```diff
@@ -284,7 +284,7 @@ def process_result(self, output, msg):
 
       if output_bytes is not None:
         props = {"__pfn_input_topic__" : str(msg.topic), "__pfn_input_msg_id__" : base64ify(msg.message.message_id().serialize())}
-        self.producer.send_async(output_bytes, partial(self.done_producing, msg.consumer, msg.message, self.producer.topic()), properties=props)
+        self.producer.send_async(output_bytes, partial(self.done_producing, msg.consumer, msg.message, self.producer.topic()), properties=props, partition_key=msg.message.partition_key())
```

Comment:
we are using the `partition_key` from the input message as the `partition_key` for the output message. it doesn't seem to be correct to me. 

any reason why you are setting this key here?

Reply:
To be consistent with Java, but I will remove this for now